utils/check_variables.py

```diff
@@ -0,0 +1,10 @@
+import os
+
+def check_variables_are_defined(variables):
+  for var in variables:
+    if var not in os.environ:
```

Comment:
```suggestion
  value = os.environ.get(variable)
    if value is None:
```
We can analyze if this is a better solution, since `if var not in os.environ:` is looping for all environment variables, maybe `value = os.environ.get(variable)` is more optimal. We could discuss this.